warzone/Cards.cpp

```diff
@@ -520,6 +520,16 @@ void Deck::printDrawpile()
 	cout << "----------------" << endl;
 }
 
+	// Hand functions
+	/**
+	 * Constructor with reference to the Deck
+	 * 
+	 * \param deck
```

Comment:
This constructor does not take an argument.

```suggestion
	 * Default Constructor
```